templates/workspaces/base/terraform/network/providers.tf

```
@@ -5,5 +5,9 @@ terraform {
       source  = "hashicorp/azurerm"
       version = ">= 3.8.0"
     }
+    null = {
+      source  = "hashicorp/null"
+      version = "=3.2.0"
```

You're missing a lock file update in this PR. Also:
```suggestion
      version = "=3.2.1"
```